gr-uhd/lib/usrp_block_impl.cc

```diff
@@ -207,7 +207,7 @@ bool usrp_block_impl::_wait_for_locked_sensor(
     else {
       first_lock_time = boost::system_time(); //reset to 'not a date time'
 
-      if (boost::get_system_time() > (start + boost::posix_time::seconds(LOCK_TIMEOUT))){
+      if (boost::get_system_time() > (start + boost::posix_time::seconds(static_cast<long>(LOCK_TIMEOUT)))){
```

Comment:
whereas the instances above a mere annoyances on a logic level, this is a real bug, because there's a serious difference between locking for 1.5s and 1.0s. CC: @mbr0wn